src/app/server/RendezvousServer.cpp

```diff
@@ -62,15 +62,6 @@ void RendezvousServer::OnPlatformEvent(const DeviceLayer::ChipDeviceEvent * even
     {
```

Comment:
This PR was only closing the BLE connections after the OperationalNetwork is Enabled. Not when it was commissioned. 

Is it possible there's a different problem in the python controller?

Reply:
Or maybe there's a timing issue here?

Reply:
Why is this unique to python and/or raspi? The proposed change makes semantic sense about when to stop advertising in the SDK... Is this a race we need to account for in the SDK? Does it need to move? Is this a platform issue?

Reply:
Also as a comment: @sagar-apple, sounds like we're missing tests here, and we should have added tests that covered this critical use case in the SDK that confirm that the device is no longer advertising BLE.